libs/minecraft/InventoryUtils.arucas

```diff
@@ -0,0 +1,301 @@
+import * from minecraft.MinecraftUtils;
+import * from util.Collections;
+class InventoryUtil {
+    var player;
+    var debug = false;
+    var selectedSlot = null;
+    var preserveDurability = false;
+    var pointer = 0;
+    var hotbarSlotCache = {};
+    //class InventoryUtil does not offer static method, instead, it offers player-related options based on binded player entity
```

Comment:
But why? This is just weird. These helper methods should be static. I mean after all there is only one player instance anyways.